functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/JobActivityHistorySpec.groovy

```diff
@@ -0,0 +1,94 @@
+
+package org.rundeck.tests.functional.selenium.jobs
+
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.WebDriverWait
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.execution.ExecutionShowPage
+import org.rundeck.util.gui.pages.jobs.JobListPage
+import org.rundeck.util.gui.pages.activity.ActivityPage
+import org.rundeck.util.common.jobs.JobUtils
+import java.time.Duration
+
+@SeleniumCoreTest
+class JobActivityHistorySpec extends SeleniumBase {
+    WebDriverWait wait
+    String jobId
+
+    /**
+     * Setup the project for the test.
+     */
+    def setupSpec() {
+        setupProject(SELENIUM_BASIC_PROJECT)
+    }
+
+    /**
+     * Setup the test environment.
+     */
+    def setup() {
+        // Set up WebDriverWait to wait up to 30 seconds for elements that take time to appear or change
+        wait = new WebDriverWait(getDriver(), Duration.ofSeconds(30))
+        def loginPage = go LoginPage
```

Comment:
nit:

Note: My `nit` comments are FYIs, subjective and will not stop the PR from getting approved! :-) 

In general, it's considered too noisy to introduce a var to use it *once* right away.
This could be compressed to something like `(go LoginPage).login(TEST_USER, TEST_PASS)`